design/requirements/_template_requirement.md

```diff
@@ -0,0 +1,18 @@
+# Requirement Title
+The <navigation system> should be able to <shall> <do something>
+
+## More details
+- Why is this needed?
+- What is the expected user interaction? 
+- What use case does this map to?
+
```

Comment:
Are there any non-functional requirements associated with this functional requirement?